backtrace-library/src/main/java/backtraceio/library/base/BacktraceBase.java

```diff
@@ -546,4 +585,8 @@ private void addReportAttachments(BacktraceReport report) {
             }
         }
     }
+
+    private boolean isBreadcrumbsEnabled() {
+        return database != null && database.getBreadcrumbs() != null;
```

Comment:
i would rather call it isBreadcrumbAvailable instead.

Reply:
good point